docs/spec/spec.md

```diff
@@ -166,6 +166,17 @@ The conforming implementation of the specification is released and included in t
         * 13.1.3 [The `subgraph:ReferenceResolver`](#1311-the-referenceresolver)
 14. [Tools](#14-tools)
     * 14.1 [GraphiQL Client](#141-graphiql-client)
+15. [Experimental Features](#15-experimental-features)
+    * 15.1. [DataLoader](#151-dataloader)
+        * 15.1.1. [DataLoader API](#1511-dataloader-api)
+            * 15.1.1.1. [The `load` method](#15111-the-load-method)
+            * 15.1.1.1. [The `load` method](#15111-the-load-method)
+            * 15.1.1.2. [The `dispatch` method](#15113-the-dispatch-method)
+        * 15.1.2. [Engaging DataLoaders](#1512-engaging-dataloaders)
+            * 15.1.2.1. [`loadXxx` Method Mapping](#15121-loadxxx-method-mapping)
+            * 15.1.2.2. [The `@dataloader:Loader` Annotation](#15122-the-dataloaderloader-annotation)
+            * 15.1.2.3. [The Batch Load Function](#15123-the-batch-load-function)
+            * 15.1.2.4. [The `map<dataloader:DataLoader>` Parameter](#15124-the-mapdataloaderdataloader-parameter)
```

Comment:
```suggestion
        * 15.1.2 [Engaging DataLoaders](#1512-engaging-dataloaders)
            * 15.1.2.1 [`loadXxx` Method Mapping](#15121-loadxxx-method-mapping)
            * 15.1.2.2 [The `@dataloader:Loader` Annotation](#15122-the-dataloaderloader-annotation)
            * 15.1.2.3 [The Batch Load Function](#15123-the-batch-load-function)
            * 15.1.2.4 [The `map<dataloader:DataLoader>` Parameter](#15124-the-mapdataloaderdataloader-parameter)
```
Do we need `.` here?

Reply:
No can be removed

Reply:
Fixed: [8b5b8b2](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1505/commits/8b5b8b22197b314c3501d1a34bd049e975a678bd)